src/js/components/TextArea/TextArea.js

```diff
@@ -48,11 +44,13 @@ const TextArea = forwardRef(
             if (onBlur) onBlur(event);
           }}
           onChange={event => {
+            const nextValue = event.target.value;
             if (formContext && name) {
-              formContext.set(name, event.target.value);
+              formContext.set(name, nextValue);
+              setValue(nextValue);
             }
             if (onChange) {
-              onChange(event);
```

Comment:
We still need to call `onChange`, in case the user passes in an `onChange`.

Reply:
gotcha, was just going to ask about this change